app/views/content_pages/_details_for_a_single_page.erb

```diff
@@ -4,12 +4,12 @@
     <p class="govuk-heading-m">
       <%= page_to_render.title %>
       <span class="links">
-        <%= link_to 'Versions', versions_content_page_path(page_to_render), { :class => "govuk-link" } %>
+        <%= link_to 'Versions', versions_content_page_path(page_to_render), { class: "govuk-link" } %>
         <% if !page_to_render.parent_id %>
-          |<%= link_to 'Add a child page', new_content_page_path(:parent_id => page_to_render.id) %>
+          |<%= link_to 'Add a child page', new_content_page_path(:parent_id => page_to_render.id), { class: "govuk-link" } %>
```

Comment:
Very minor - which you can ignore - but is it work updating the `parent_id` assignment:
```ruby
|<%= link_to 'Add a child page', new_content_page_path(parent_id: page_to_render.id), { class: "govuk-link" } %>
```